apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog.Snippets/nlog-jsonTemplate.xml

```diff
@@ -0,0 +1,72 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
This file looks excessively large to me, for a snippet. The intention would be for it to appear in a page like this: https://googleapis.dev/dotnet/Google.Cloud.Logging.NLog/latest/index.html

I very much doubt that it's worth having a 9-level nested layout just to show how to use this.